sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -4187,6 +4188,11 @@ object GpuOverrides extends Logging {
       ExprChecks.projectOnly(TypeSig.STRING, TypeSig.STRING),
       (a, conf, p, r) => new GpuUuidMeta(a, conf, p, r)
     ),
+    expr[StaticInvoke](
+      desc = "StaticInvoke",
+      StaticInvokeCheck,
+      (a, conf, p, r) => new StaticInvokeMeta(a, conf, p, r)
+    ),
```

Comment:
```suggestion
    ).note("The supported types are indeterministic since it's a dynamic expression"),
```

Please add the above note, then the `supportedExprs.csv` files will shows like:

```
StaticInvoke,S, ,The supported types are indeterministic since it's a dynamic expression,project,result,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS,PS
```